README.md

```diff
@@ -125,6 +125,40 @@ class ViewController: UIViewController {
 }
 ```
 
+### KeyboardAutoNavigator - when using a KeyboardToolbar
+Create a `KeyboardToolbar`, configuring it with back/next/done buttons as appropriate. Then, create a `KeyboardAutoNavigator`, passing in your  toolbar. The position of the text inputs determines the navigation order for traversing from one to the next. Optionally, implement `KeyboardAutoNavigatorDelegate` to receive call backs when tapping "Back", "Next", and "Done" in your `KeyboardToolbar`.
```

Comment:
Extra space between "your" and "toolbar" in the second sentence.

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/f0fb818e19adc3fe77eecb915f01f9ef479cc927